clients/src/test/java/org/apache/kafka/clients/producer/internals/StickyPartitionCacheTest.java

```diff
@@ -81,8 +82,8 @@ public void unavailablePartitionsTest() {
             new PartitionInfo(TOPIC_C, 0, null, NODES, NODES)
         );
         
-        Cluster testCluster = new Cluster("clusterId", asList(NODES[0], NODES[1]), allPartitions,
-            Collections.<String>emptySet(), Collections.<String>emptySet());
+        Cluster testCluster = new Cluster("clusterId", asList(NODES[0], NODES[1], NODES[2]), allPartitions,
+            Collections.emptySet(), Collections.emptySet());
```

Comment:
@jolshan @cmccabe Please review this change to the StickyPartitionCcaheTest.

Reply:
It seems that those extra <String> lines are in DefaultPartitionerTest as well. (That is where I got the line from.)
Also, this is more of a question for myself to understand, but what is the fourth node for?